### DISCUSSION TITLE
Trinkets > Curio bridge

### DISCUSSION THREAD

**Discussion by nebulazorua** [Sinytra/ConnectorExtras#3] [Ideas]
I'm not sure how Trinkets or Curios work under the hood, as I'm not well versed in Java nor Minecraft modding, but if possible then a bridge for mods that use Trinkets to use Curio and vice versa would be nice

**Comment by walksanatora** [comment]
that would be nice

**Comment by willy-wigga** [comment]
yes, this would be good for "Scout" mod

**Comment by walksanatora** [comment]
okay so I know this is VERY sideliney... but why not instead just use
https://modrinth.com/mod/accessories with the relevant [trinkets](https://modrinth.com/mod/accessories-tc-layer) or [curios](https://modrinth.com/mod/accessories-cc-layer) compat layer
